src/backend/src/services/users.services.ts

```diff
@@ -339,4 +339,27 @@ export default class UsersService {
 
     return userScheduleSettingsTransformer(scheduleSettings);
   }
+
+  static async setUserScheduleSettings(
```

Comment:
Can you check that emails are unique? See how its done for phone numbers in setUserSecureSettings.